src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/sessions/SessionTemplateDto.kt

```diff
@@ -27,8 +27,8 @@ data class SessionTemplateDto(
   val sessionDateRange: SessionDateRangeDto,
   @Schema(description = "visit type", example = "SOCIAL", required = true)
   val visitType: VisitType,
-  @Schema(description = "biWeekly", example = "true", required = true)
-  val biWeekly: Boolean,
+  @Schema(description = "number of weeks until the weekly day is repeated", example = "1", required = true)
+  val weeklyFrequency: Int,
```

Comment:
we could add a @Min(0) validation to avoid lower than 0 frequency or change variable to UInt. The validation might be better option. Also is weeklyFrequency of 0 allowed?

Reply:
It would be @min(1)